CONTRIBUTING.md

```diff
@@ -0,0 +1,149 @@
+# How can I contribute?
+
+- You can fix the typos (if any) in any documentation of this repo.
+- You can check for any **unassigned** [issues](https://github.com/Real-Dev-Squad/website-www/issues) and **comment** on that issue that you'd like to get that issue assigned to you. (Remember: Do not work on issues assigned to someone else and do not work on any issue without having it assigned to you.)
+- Create issue if you see any bug and then once you get approved from the admins, you can assign yourself and start working on it.
+
+# How can you make a **Pull Request** (PR)
+
+**Note**: Steps 1, 2 and 3 are **one-time** steps required for setup. If you have already cloned the repo and added upstream, consider following this documentation from step 4.
+
+1. **Forking repository**
+
+Fork this repository using the **Fork** option at the top-right corner of this page. This will create your own copy of this repository. You'll be redirected to your forked repository. Copy the link of this repository (which will look like `https://github.com/<your-username>/website-www/`) as you'll need it in the step 2.
+
+![how-to-fork](https://user-images.githubusercontent.com/53713926/212554906-0308d029-7cf6-4403-8f7b-0ca4d50de46f.png)
+
+
+
+2. **Cloning repository and Installing packages**
+
+Clone your forked repository, this will download your copy of repository in your computer. To do this, open your terminal (command prompt/bash/git bash) and enter the following command, paste your link after the word **clone** without the **<>**.
+
+```
+git clone <link which you copied in the step 1>
+```
+
+Once you have cloned the repository, now you should go into the folder containing the repository. You can do that with:
+
+```
+cd website-my
+```
+
+Now you should install the required packages to run this project by doing:
+
+```
+yarn or yarn install
+```
```

Comment:
This we have to do after we checkout to `develop-ember` because this doc is for migration purposes and `develop` and `develop-ember` contain two different codebases, so installing packages before checking out to that codebase will not make sense.